src/Core/Services/Implementations/RelayPushNotificationService.cs

```diff
@@ -35,7 +34,6 @@ public class RelayPushNotificationService : BaseIdentityClientService, IPushNoti
         {
             _deviceRepository = deviceRepository;
             _httpContextAccessor = httpContextAccessor;
-            _logger = logger;
```

Comment:
I take it this was unused?

Reply:
Yep, `BaseIdentityClientService` handles logging